path_of_titans/README.md

```diff
@@ -0,0 +1,40 @@
+# Path Of Titans
+
+<!--Please remove these comments and irelevent parts for the server egg your adding before summiting a PR request-->
+
+## From their [Github](https://github.com/Hendalf-DEV/eggs)
```

Comment:
Wy does this link to you? This is not ment for that. It is meant to link to the server repo if their is one.

Reply:
Removed.